bliss/encoder/metrics.py

```diff
@@ -138,36 +138,31 @@ def get_report_bins(self):
 
 
 class FluxBinMetric(GeneralBinMetric):
-    def __init__(self, base_nmgy_bin_cutoffs: list, report_bin_unit: str, exclude_last_bin: bool):
-        super().__init__(bin_cutoffs=base_nmgy_bin_cutoffs, exclude_last_bin=exclude_last_bin)
+    def __init__(self, base_njy_bin_cutoffs: list, report_bin_unit: str, exclude_last_bin: bool):
+        super().__init__(bin_cutoffs=base_njy_bin_cutoffs, exclude_last_bin=exclude_last_bin)
 
-        self.base_nmgy_bin_cutoffs = self.bin_cutoffs
+        self.base_njy_bin_cutoffs = self.bin_cutoffs
         self.report_bin_unit = report_bin_unit
-        assert self.report_bin_unit in {"mag", "nmgy", "njymag"}, "invalid bin type"
+        assert self.report_bin_unit in {"njy", "ab_mag"}, "invalid bin type"
 
     def get_state_for_report(self, state_name):
         state = getattr(self, state_name)
         match self.report_bin_unit:
-            case "nmgy":
+            case "njy":
                 pass
-            case "mag":
-                state = torch.flip(state, dims=(-1,))
-            case "njymag":
+            case "ab_mag":
                 state = torch.flip(state, dims=(-1,))
             case _:
                 raise NotImplementedError()
         return state[..., :-1] if self.exclude_last_bin else state
 
     def get_report_bins(self):
-        report_bins = torch.tensor(self.base_nmgy_bin_cutoffs)
+        report_bins = torch.tensor(self.base_njy_bin_cutoffs)
         match self.report_bin_unit:
-            case "nmgy":
+            case "njy":
                 pass
-            case "mag":
-                report_bins = convert_nmgy_to_mag(report_bins)
-                report_bins = torch.flip(report_bins, dims=(-1,))
-            case "njymag":
-                report_bins = convert_nmgy_to_njymag(report_bins)
+            case "ab_mag":
```

Comment:
Maybe call this "mag" rather than "ab_mag"? Apparently SDSS magnitudes aren't exactly ab magnitudes

Reply:
change it to `mag`